tools/go-cli/go-whisk-cli/main.go

```diff
@@ -84,7 +91,8 @@ func main() {
 
         // Displays usage
         if displayUsage {
-            fmt.Fprintf(os.Stderr, "Run '%v --help' for usage.\n", commands.WskCmd.CommandPath())
+            fmt.Fprintf(os.Stderr, T("Run '{{.Name}} --help' for usage.\n",
+                map[string]interface{}{ "Name" : commands.WskCmd.CommandPath()}))
```

Comment:
@dubeejw - In this case, no.  %v will default to %s when a string is being processed, and the CommandPath() function returns a string.